field/generator/internal/templates/element/fixed_exp.go

```diff
@@ -4,6 +4,7 @@ const FixedExp = `
 
 {{- if .SqrtQ3Mod4}}
 	{{expByAddChain "SqrtExp" .SqrtQ3Mod4ExponentData .ElementName}}
+	{{expByAddChain "SqrtExp2" .SqrtQ3Mod4ExponentData2 .ElementName}}
```

Comment:
Name is not super descriptive, specially now that the method is package public `ExpBySqrtExp2`  ?